discovery_engine_core/web-api/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "web-api"
+version = "0.1.0"
+license = "AGPL-3.0-only"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { version = "1.0.137", features = ["derive"] }
+serde_json = "1.0.81"
+tokio = { version = "1.19.2", features = ["full"] }
```

Comment:
❓ do we actually need the full `tokio`? if not we can save some space by selecting required features

Reply:
I've limited features to `macros`, if I understand correctly should be enough for now